cmd/node/main.go

```diff
@@ -173,8 +178,14 @@ VERSION:
 		Value: math.MaxUint32,
 	}
 
-	//TODO remove uniqueID
-	uniqueID = ""
+	wd, _ = os.Getwd()
```

Comment:
treat the error.

Reply:
really don't know how to do that... it's in the var declaration...

Reply:
moved to main